tesseract_ros/tesseract_examples/tesseract_ros_examples/test/basic_cartesian_example_unit.cpp

```diff
@@ -0,0 +1,31 @@
+#include <tesseract_common/macros.h>
+TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
+#include <ros/ros.h>
+#include <gtest/gtest.h>
+TESSERACT_COMMON_IGNORE_WARNINGS_POP
+
+#include <tesseract_ros_examples/basic_cartesian_example.h>
+
+using namespace tesseract_ros_examples;
+
+TEST(TesseractROSExamples, BasicCartesianCppExampleUnit)
+{
+  ros::NodeHandle nh;
+  BasicCartesianExample example(nh, false, false, 5, "cpp");
```

Comment:
I suppose there's no reason you couldn't have a test with plotting_ true. That would catch things like when plotting broke when using time param.